beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/README.md

```diff
@@ -1,2 +1,106 @@
+
+A high-level overview of the pipeline in addition to tool parameters are available on the [GDC Documentation site](https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/). 
+
+Overall, the pipeline converts reads (CRAM or BAM) to FASTQ and (re)aligns them to the latest human reference genome. Each read group is aligned separately. Read group alignments that belong to a single sample are then merged and duplicate reads are flagged for downstream variant calling. 
+
+
+## Set-up
+
+### Workflow Installation and Requirements
+
+The workflow is written in the Workflow Description Language WDL and can be downloaded by cloning the [warp repository](https://github.com/broadinstitute/warp/tree/master) in GitHub. The workflow can be deployed using [Cromwell](https://github.com/broadinstitute/cromwell), a GA4GH compliant, flexible workflow management system that supports multiple computing platforms. 
+
+For the latest workflow version and release notes, please see the [changelog](https://github.com/broadinstitute/warp/blob/master/beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.changelog.md).
+
+### Software Version Requirements
+
+* GATK 4.0.7
+* Picard 2.18.11 (Custom Docker is used to run software on Cromwell 52)
+* Samtools 1.11
+* Python 3.0
+* Cromwell version support
+    * Successfully tested on v52
+    * Does not work on versions < v23 due to output syntax
+* Papi version support
+	* Successfully tested on Papi v2
+
+### Input Descriptions
+
+The table below describes each of the GDC pipeline inputs. The workflow requires a **single** aligned CRAM or BAM file, or a **single** unmapped BAM (uBAM) as input.
+
+| Input name | Description | Type | 
+| --- | --- | --- |
+| input_cram | A single mapped CRAM file; alternatively input can be a mapped BAM (`input_bam`) or unmapped BAM (uBAM; `ubam`) | File |
+| input_bam | A single mapped BAM file; alternatively input can be a CRAM (`input_cram`) or uBAM (`ubam`) | File |
+| cram_ref_fasta | The reference file that was used to align the CRAM input (if used); if unspecified, the workflow will use the `ref_fasta` input file by default | File |
+| cram_ref_fasta_index | CRAM reference FASTA index if CRAM is used as workflow input | File |
+| output_map | Tab-separated file containing two columns: a list of all the read group IDs found in the input_cram (or input_bam) and a list of the desired name of the uBAMs generated for read group | File |
+| unmapped_bam_suffix | Optional string used to name the output uBAM file | String |
+| ubam | A single uBAM file; alternatively, input can be a mapped BAM (`input_bam`) or CRAM (`input_cram`) | File |
+| ref_fasta | Reference FASTA used to convert an input CRAM to BAM; if CRAM is not used, this file does not need to be specified | File | 
+| ref_fai | Reference FASTA index for the `ref_fasta` input | File |
+| ref_dict | BWA reference dictionary used for alignment| File |
```

Comment:
Given that this is a different version of the reference and it it publicly available for use in the google managed bucket, should we include links to those are recommended inputs?

Reply:
Or perhaps linking to our example inputs shows the different reference? But I was coming here to comment about highlighting the different reference too, I'll comment in main thread.

Reply:
I wrote both a separate section and linked to the inputs file.